core/src/Streamly/Internal/Data/Stream/StreamK.hs

```diff
@@ -49,8 +49,8 @@
 module Streamly.Internal.Data.Stream.StreamK
     (
     -- * The stream type
-      Stream(..) -- XXX stop exporting this
-    , StreamK
+      Stream
```

Comment:
Why are we exporting `Stream` from here?

Reply:
for backward compatibility.